### PR TITLE
Release notes for Android release 1.0.99

### PR DESCRIPTION
**Pull request by srirambv**
Fixes: #1751

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,13 @@
...
-  - New bookmarks not being sync'd ([#1702](https://github.com/brave/browser-android-tabs/issues/1702))
-  - Android device getting removed from sync chain ([#1354](https://github.com/brave/browser-android-tabs/issues/1354))
-  - Frequent disconnect from sync chain ([#1535](https://github.com/brave/browser-android-tabs/issues/1535))
\ No newline at end of file
+- Fixed bookmarks not being synced in certain cases. ([#1702](https://github.com/brave/browser-android-tabs/issues/1702))
+- Fixed android devices randomly being removed from the sync chain. ([#1354](https://github.com/brave/browser-android-tabs/issues/1354))
+- Fixed android devices randomly being disconnected from the sync chain. ([#1535](https://github.com/brave/browser-android-tabs/issues/1535))
+- Upgraded Chromium to 75.0.3770.89. ([#1681](https://github.com/brave/browser-android-tabs/issues/1681))
+
+## [1.0.99](https://github.com/brave/browser-android-tabs/releases/tag/v1.0.99)
```
Comment (LaurenWags) [inline CHANGELOG.md]: on desktop we list newest version and notes at the top of the doc, for consistency across platforms should probably do that here as well.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,13 @@
...
+
+ - Fixed "Set as default browser" option from the settings menu. ([#1495](https://github.com/brave/browser-android-tabs/issues/1495))
```
Comment (LaurenWags) [inline CHANGELOG.md]: I'd probably say:
Removed "Set as default browser" option from settings menu when Brave is already default browser.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,8 +2,13 @@
...
+ - Fixed forward button icon for mobile. ([#911](https://github.com/brave/browser-android-tabs/issues/911))
```
Comment (LaurenWags) [inline CHANGELOG.md]: I'd probably say:
Changed forward button icon for mobile phones.

**Comment by samartnik** [comment]
@srirambv could you move these changes to `base-75.0.3770.101` branch?

**Comment by srirambv** [comment]
Closing in favour of #1756

**Pull request closed without merging**